contracts/test/staking/single-asset-staking.js

```diff
@@ -157,11 +157,11 @@ describe("Single Asset Staking", function () {
     ).to.be.revertedWith("Only token contract can make this call");
 
     // This generate the data needed for calling stakeWithSender
-    const interface = ognStaking.interface;
```

Comment:
Is this abbreviation a common convention?

Reply:
iface?   No idea, but `interface` is a reserved word in JS, hence the change.

Reply:
@mikeshultz Ah I see! Thank you for explaining that. How was that able to work before if `interface` is a reserved word?

Reply:
I could be wrong, but I think `interface` was introduced in ES6, which probably explains why we haven't see any issues.  I changed it out of an abundance of caution and to prevent any future conflicts.